vagrant/fedora-26/first_time_setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# This script is to be run after `vagrant up` failing
+# with failure to setup shared folders.
+#
+# It downloads and installs Virtual Box Guest Additions.
```

Comment:
See the README in the parent folder. If you have the vagrant-vbguest plugin installed then the Guest Additions will be installed before the bootstrap.sh script is run during provisioning, and there is no need to do anything extra for it.

Reply:
I've missed it!

I will now close this PR because you already have it on master.

Thanks again!